### DISCUSSION TITLE
phpoffice/phpspreadsheet(1.15.0):

### DISCUSSION THREAD

**Discussion by 477dimmur253** [YetiForceCompany/YetiForceCRM#14103] [How to] [answered]
Hello,
I keep getting an e-mail notification about libraries vulnerabilities regarding **phpoffice/phpspreadsheet(1.15.0)**. You can't update it automatically, only manually.
I have replaced the files on the server with the new version and the notification still keeps coming every few hours. 
Do I have to replace something else too or do some additional thing?

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
it requires updating composer
